src/styles/components/layout/inputs/RadioInput/_RadioInput.scss

```
@@ -0,0 +1,73 @@
+.input-radio {
+  -moz-appearance: none;
+  -webkit-appearance: none;
```

```suggestion
  appearance: none;
```
https://caniuse.com/?search=appearance

---

Yes ! Je m'étais arrêté au premier CSS appearance sur caniuse, je n'avais pas vu que pour la valeur none il n'y avait pas besoin de préfixe. Bien vu merci !